src/Microsoft.Graph.Core/Requests/IBaseRequest.cs

```diff
@@ -12,6 +12,11 @@ namespace Microsoft.Graph
     /// </summary>
     public interface IBaseRequest
     {
+        /// <summary>
+        /// Gets or sets the response handler for the request.
+        /// </summary>
+        IResponseHandler ResponseHandler { get; set; }
```

Comment:
This is going to be a breaking change for anyone who has implemented their own Request object that implements IBaseRequest.

Reply:
Thank you for catching this. I'm confident that this will break someone out there.

Reply:
Fixed, we just needed this on BaseRequest.